azure-pipelines.yml

```diff
@@ -8,20 +8,14 @@ trigger:
 
 strategy:
   matrix:
-    linux_node_8:
-      imageName: 'ubuntu-18.04'
-      node_version: 8.x
-    linux_node_10:
-      imageName: 'ubuntu-18.04'
-      node_version: 10.x
     linux_node_12:
-      imageName: 'ubuntu-18.04'
+      imageName: 'ubuntu-20.04'
       node_version: 12.x
```

Comment:
We shouldn't test with 12.x then, right?

Reply:
good point